src/effects/effectslot.cpp

```diff
@@ -249,12 +249,12 @@ void EffectSlot::loadEffectFromPreset(const EffectPresetPointer pPreset) {
     }
     EffectManifestPointer pManifest = m_pBackendManager->getManifest(
             pPreset->id(), pPreset->backendType());
-    loadEffectInner(pManifest, pPreset, true);
+    loadEffectInner(std::move(pManifest), pPreset, true);
 }
 
 void EffectSlot::loadEffectWithDefaults(const EffectManifestPointer pManifest) {
     EffectPresetPointer pPreset = m_pPresetManager->getDefaultPreset(pManifest);
-    loadEffectInner(pManifest, pPreset, false);
+    loadEffectInner(pManifest, std::move(pPreset), false);
```

Comment:
above you move pManifest ... ? 
I think pPreset should be passed as raw pointer of const reference, because it is only borrowed.